projects/openshift.py

```diff
@@ -61,10 +67,57 @@ def deploy():
 
 
 @task
+def build():
+    with cd('origin'):
+        run('make clean build')
+
+
+@task
 def start():
+    build()
+
     with cd('origin'):
-        run('make')
-        sudo('nohup ./_output/local/go/bin/openshift start &')
+        run('test -e {0} || {1} start master '
+            '--write-config=openshift.local.config'.
+            format(MASTER_YAML_FILE, OPENSHIFT))
+
+        run('mkdir -p ~/.config/openshift/')
+        run('cp {0} ~/.config/openshift/config'.format(ADMIN_KUBECONFIG_FILE))
+
+        # used the sleep workaround due to fabric issue #395
+        sudo('nohup {0} start master --config={1} & sleep 5; exit 0'.
```

Comment:
Have you tried "pty=false" as other possible solution? I was actually curious to know if something like this would work:

    sudo('{0} start master --config={1} </dev/null &>openshift.log &'.format(OPENSHIFT, MASTER_YAML_FILE), pty=false)